src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -322,6 +330,7 @@ public function getPreferredModulesAction()
      */
     public function importModuleAction(Request $request)
     {
+        $translator = $this->get('translator');
```

Comment:
In different place of this file we have:
` $translator = $this->container->get('translator');`

Is there any reason for that?

Also, maybe we could create `getTranslator` method and do `$this->getTranslator()->trans` instead of having to type this line of code every time? :)